backend/tournesol/models/comparisons.py

```diff
@@ -110,6 +110,23 @@ def get_comparison(user, poll_id, uid_a, uid_b):
 
         return comparison, True
 
+    def mark_compared_entities_as_seen(self):
+        from tournesol.models import ContributorRating  # pylint: disable=import-outside-toplevel
+
+        ratings = ContributorRating.objects.filter(
+            poll=self.poll,
+            user=self.user,
+            entity__in=[self.entity_1, self.entity_2],
+        )
+
+        for rating in ratings:
+            rating.entity_seen = True
+
+        ContributorRating.objects.bulk_update(
+            ratings,
+            ["entity_seen"],
+        )
```

Comment:
Can be simplified into a single query with:
```suggestion
        ContributorRating.objects.filter(
            poll=self.poll,
            user=self.user,
            entity__in=[self.entity_1, self.entity_2],
        ).update(entity_seen=True)
```